src/PrimeForm.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { useState } from 'react';
+import { observer } from 'mobx-react-lite';
+import { Button, Text, Box } from 'rebass';
+import { store } from './store';
+import styled from 'styled-components';
+import BigNumber from 'bignumber.js';
+import { Prime } from './types';
+
+const PrimeInput = styled.input`
+  font-size: 14px;
+  height: 34px;
```

Comment:
I would recommend avoid setting height as explicit value and use padding and fons-size properties instead to size input.
Cause it might cause some issues with inherited line-height (not our case, but still ^__^)